lesson7/index.html

```diff
@@ -6,8 +6,13 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" type="text/css" />
     <title>TerraceTechフロントエンドエンジニア養成所</title>
+    <link rel="stylesheet" href="/style.css">
   </head>
   <body>
+    <div id="loader">
+      <img src="img/loading-circle.gif" alt="">
+    </div>
+
```

Comment:
Since the image will be displayed in the asynchronous process, I think it is better to create it in JS.

Reply:
Thank you. I recreated them in JS and implemented div of image element as well.
[23f90cc](https://github.com/yuka830/js-lessons/pull/9/commits/23f90ccb9b9230c1f795760c6f6033aa1f6b639d)

Reply:
> I declared the constant loader and loaderImage in window.onload because I thought they are rarely to be used elsewhere. Is this idea correct?

I'm pretty sure it's not defined globally.
However, the process here can be cut out to a function. I think it is better to extract as much as possible that can be extracted as a function and execute it where it is used.

1. Processing while loading
2. Processing when the loading is finished